### PR TITLE
Feature/plcn 203 create nested contact person deal

### PR DESCRIPTION
**Pull request by s-bogelund**

### CODE REVIEW

File: src/components/wall-of-clients/NestedContactPerson.tsx [hunk 1]
```diff
@@ -14,46 +15,49 @@ import DealsStatusSummary from '../summaries/DealsStatusSummary';
...
 		FRAGMENT_ACCOUNT_MANAGERFragment[]
 	>([]);
 	const [suppliersState, setSuppliersState] = useState<FRAGMENT_SUPPLIERFragment[]>([]);
+	const contactArray = [contact];
 
 	useEffect(() => {
 		setDealsState(
 			removeArrayDuplicates(contact.dealContacts.flatMap(dealContact => dealContact.deal))
 		);
 	}, [contact]);
```
[the remark this answers is not in the document]
Comment (itm-mne) [reply src/components/wall-of-clients/NestedContactPerson.tsx:53]: Same goes for remaining useEffects concerning mappingof data

File: src/pages/WallOfClients.tsx [hunk 2]
```diff
@@ -102,7 +102,7 @@ const WallOfClients = () => {
 					</PopupFilterWrapper>
 				</SecondaryFilterContainer>
 			</FilterContainer>
-			{loading && <CompanyCardsSkeleton numSkeletons={5} />}
+			{loading && <CompanyCardsSkeleton numSkeletons={10} />}
```
Comment (itm-mne) [inline src/pages/WallOfClients.tsx:105]: Should we have another Skeleton component for when we are loading paginated data? Maybe a component with fewer skeleton lines? If so, please create a JIRA task.

**Review by itm-mne — CHANGES_REQUESTED** [review]
Move mapping from useEffects into functions for NestedContactPerson.tsx

**Review by AntonEsbjergIT-Minds — COMMENTED** [review]
The nesting indicator is not visible in lightmode.
Reddit line cannot be seen in lightmode
And the dividing line in the summaries is very faint in lightmode

**Pull request merged**